capsules/src/signbus/SignbusLayers.md

```diff
@@ -0,0 +1,87 @@
+# Signbus Communication Layers
+
+
+-\ USERLAND <br />
+&nbsp;&nbsp;  -\ app_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;    -\ protocol_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    -\ io_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;        -\ port_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;        -\ I2C DRIVER <br />
```

Comment:
Some unicode and a fenced code block will make this look nice in raw text and rendered form:

    ```
    USERLAND
     ↳ app_layer
      ↳ protocol_layer
       ↳ io_layer
        ↳ port_layer
         ↳ I2C DRIVER
    ```